selinux/ipa.te

```diff
@@ -412,3 +420,28 @@ optional_policy(`
 optional_policy(`
        systemd_private_tmp(ipa_custodia_tmp_t)
 ')
+
+optional_policy(`
+    gen_require(`
+        type tomcat_t;
+    ')
+    can_exec(tomcat_t, ipa_pki_retrieve_key_exec_t)
+    pki_manage_tomcat_etc_rw(ipa_pki_retrieve_key_t)
+')
+
+optional_policy(`
+    gen_require(`
+        type devlog_t;
+    ')
+
+    dontaudit ipa_custodia_t devlog_t:lnk_file read_lnk_file_perms;
+')
+
+optional_policy(`
+    java_exec(ipa_custodia_pki_tomcat_t)
+    # allow Java to read system status and RNG
+    dev_read_urand(ipa_custodia_t)
+    dev_read_rand(ipa_custodia_t)
+    kernel_read_network_state(ipa_custodia_t)
+    dev_read_sysfs(ipa_custodia_t)
```

Comment:
Interfaces for kernel modules do not need to be in an optional module. Was your intent to make them active only together with java?
Anyway, alphabetical order makes it more readable.

Reply:
@zpytela That means we can keep this if the items are ordered?